scripts/restart_video.sh

```diff
@@ -7,11 +7,13 @@ fi
 # this is followed up in start_video.sh
 cp /home/pi/vidformat.param /home/pi/vidformat.param.bak
 
-echo $1 > /home/pi/vidformat.param
-echo $2 >> /home/pi/vidformat.param
-echo $3 >> /home/pi/vidformat.param
-echo $4 >> /home/pi/vidformat.param
+if [ "$#" == 4 ]; then
+        echo $1 > /home/pi/vidformat.param
+        echo $2 >> /home/pi/vidformat.param
```

Comment:
8 spaces indentation?

Reply:
From the previous indentation level of the same file, we should uniform this in all files in future PRs.